frontend/javascripts/oxalis/view/components/categorization_label.js

```diff
@@ -63,6 +69,7 @@ export function CategorizationSearch({ searchTags, setTags, localStorageSavingKe
           onClose={() => {
             removeTag(tag);
           }}
+          style={{ cursor: "pointer" }}
```

Comment:
I see the advantage of having the `pointer` cursor for the tags next to the datasets, since they are clickable, but for the tags which are already selected and shown at the top, the cursor should just be `default` since they are not clickable (except for the close button which has the `pointer` cursor).

```suggestion
          style={{ cursor: "default" }}
```

Reply:
ah, of course, you are right!